data_pipelines_cli/cli_commands/dbtcloud.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+import click
+from typing import Any, Dict
+from ..cli_constants import BUILD_DIR
+
+from data_pipelines_cli.dbt_cloud_api_client import DbtCloudApiClient
+from ..cli_utils import echo_info
+from ..config_generation import read_dictionary_from_config_directory
+
+
+def read_dbtcloud_config(env: str) -> Dict[str, Any]:
+    """
+    Read dbt Cloud configuration.
+
+    :param env: Name of the environment
+    :type env: str
+    :return: Compiled dictionary
+    :rtype: Dict[str, Any]
+    """
+    return read_dictionary_from_config_directory(BUILD_DIR.joinpath("dag"), env, "dbtcloud.yml")
+
+
+@click.command(name="configure-cloud", help="Create dbt Cloud project")
+@click.option(
+    "--account_id",
+    type=int,
+    required=True,
+    help="""dbt Cloud Account identifier To obtain your dbt Cloud account ID, sign into dbt Cloud in your browser.
+     Take note of the number directly following the accounts path component of the URL - this is your account ID""",
+)
+@click.option(
+    "--token",
+    type=str,
+    required=True,
+    help="API token for your DBT Cloud account."
+         "You can retrieve your User API token from your User Profile (top right icon) > API Access."
+         "You can also use Service Token. Retrieve it from Account Settings > Service Tokens > Create Service Token.",
+)
+@click.option(
+    "--remote_url",
+    type=str,
+    required=True,
+    help="Git stores remote URL"
+         " Note: After creating a dbt Cloud repository's SSH key, you will need to add the generated key text as"
+         " a deploy key to the target repository. This gives dbt Cloud permissions to read / write in the repository."
+)
+@click.option(
+    "--keyfile",
+    type=str,
+    required=True,
+    help="Bigquery keyfile"
+)
+@click.option(
+    "--env",
+    default="local",
+    type=str,
+    help="Name of the environment",
+    show_default=True)
+def configure_cloud_command(
+        account_id: int,
+        token: str,
+        remote_url: str,
+        keyfile: str,
+        env: str
+) -> None:
+    client = DbtCloudApiClient(f"https://cloud.getdbt.com/api", account_id, token)
+
+    dbtcloud_config = read_dbtcloud_config(env)
+    file = open(keyfile)
+    keyfile_data = json.load(file)
+    project_id = client.create_project(dbtcloud_config["project_name"])
+    (repository_id, deploy_key) = client.create_repository(project_id, remote_url)
+    echo_info("You need to add the generated key text as a deploy key to the target repository.\n"
```

Comment:
Is this message printed every time?

Reply:
We are not currently checking whether such a project already exists. Every time we run the command a new one is created, which is why we see this message every time.